src/montecarlo/run_fund_scc_mcs.jl

```diff
@@ -29,7 +29,7 @@ function run_fund_scc_mcs(trials = 10000; years = [2020], rates = [0.03], ntimes
 
     # get models and sim
     mcs = getmcs()
-    mm = create_marginal_FUND_model()
+    mm = get_marginal_model()
     models = [mm.base, mm.marginal] #fixing bug in Mimi so this can go back to models = mm
```

Comment:
@davidanthoff @ckingdon95 we should set this line back to `models = mm`, since we fixed support for that and it will run in `Mimi v9.4`, and we depend on this functionality elsewhere in the testing